image/refs/layout/tar.go

```diff
@@ -0,0 +1,149 @@
+// Copyright 2016 The Linux Foundation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package layout
+
+import (
+	"archive/tar"
+	"bytes"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io"
+	"os"
+	"strings"
+
+	casLayout "github.com/opencontainers/image-spec/image/cas/layout"
```

Comment:
On Mon, Aug 29, 2016 at 07:01:09PM -0700, Stephen Day wrote:
> > +	casLayout "github.com/opencontainers/image-spec/image/cas/layout"
> 
> avoid camelcase for package names.

I need to distinguish this from image/layout.  Do you have an
alternative local name you'd like me to use?